src/unix/cygwin.c

```diff
@@ -36,9 +36,41 @@ int uv_uptime(double* uptime) {
 }
 
 int uv_resident_set_memory(size_t* rss) {
-  /* FIXME: read /proc/meminfo? */
-  *rss = 0;
+  char buf[1024];
+  const char* s;
+  ssize_t n;
+  long val;
+  int rc;
+  int i;
+  struct sysinfo si;
+
+  rc = uv__slurp("/proc/self/stat", buf, sizeof(buf));
+  if (rc < 0)
+    return rc;
+
+  /* rss: 24th element */
+  for (s = buf, i = 1; i <= 23; i++, s++) {
+    s = strchr(s, ' ');
+    if (s == NULL)
+      goto err;
+  }
```

Comment:
For my curiosity, does this work when the executable name contains spaces? I recall the previous incarnation handled that.

Reply:
No it won't. The filename of the executable may contain both spaces and parentheses.
so finding the last occurrence of `)` would help